opportunity_job_app/public/locales/en/en.json

```diff
@@ -0,0 +1,12 @@
+{
+    "logo_title": "JOB OPPORTUNITIES",
+    "banner_heading": "Job Opportunities",
+    "banner_content": "Welcome to Job Opportunities - a platform that makes a difference in the world of employment! Our vision is simple - We want to create a space where every woman can achieve success and have equal opportunities to find dignified work. Whether you are looking for a job or offering employment opportunities, we invite you to join our platform so that we can work together to create a positive vision for the future of all women in the labor market. Job opportunities - because together we make a difference!",
+    "card1_title": "looking for a job",
+    "card1_text": "Sign up",
+    "card2_title": "Job offer",
+    "card3_user_title": "active jobs",
+    "card3_user_text": "See page",
+    "card3_company_title": "My jobs",
+    "card3_company_text": "Create a job"
+}
```

Comment:
Ako cemo sve stringove imati u jednom fajlu, predlozio bih da koristimo sablon za nestovanje koji koristimo i za nestovanje samih komponenti u kojima ce se ti stringovi nalaziti.
Npr. ako imamo komponentu (tj. fajl) `app/job-offers/page.tsx` i u njom imamo neki naslov (title), predlazem da to u ovom JSON fajlu bude:
```
{
    "app": {
        "jobOffers": {
            "page": {
                "title": "Job Offers"
            }
        }
    }
}

```
Takodje, kao sto vidis, koristio sam `camelCase`, jer ce taj JSON biti ucitan u komponenti kao JS objekat, a u JS-u je `camelCase` konvencija pisanja objekata, npr: `const personalInfo = { firstName: "Nikola" }`